HaroldAdviser/Controllers/WarningsController.cs

```diff
@@ -0,0 +1,35 @@
+﻿using HaroldAdviser.Models;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using System.Linq;
+
+namespace HaroldAdviser.Controllers
+{
+    [Route("/api")]
+    public class WarningsController : Controller
```

Comment:
please change route to smth like api/warning

Reply:
done